publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+# Results of accessibility self-review for Web of Things (WoT) Architecture
+
+Answers to accessibility (FAST) review checklist [(from here)](https://w3c.github.io/apa/fast/checklist.html)
+for the Web of Things (WoT) architecture as a whole.
+These answers are generally written to be applicable to all WoT deliverables.
+Where a response is specifically relevant to a particular deliverable that will be identified.
+
+## If technology allows visual rendering of content
+The WoT architecture does not directly support visual rendering of content.
+Some devices adhering to WoT standards may have visual display aspects, for example
+light bulbs that can be turned on or off, or small displays.
+However, WoT in general has no provision to constrain visual displays, 
+as this (and other controls limiting the behavior of devices)
+was not in the scope of our current charter.
+
+### [no] There is a defined way for a non-visual rendering to be created.	
+No.
+### [no] Content can be resized.	
+No specified mechanism.
+### [possible] Luminosity and hue contrast can adapt to user requirements.	
+No specified mechanism, although WoT TDs for lights usually have such controls, and WoT supports them.
+### [possible] Text presentation attributes can be changed.
```

Comment:
suggest n/a

Reply:
making it "possible" to be consistent as discussed.   Also added a little more detail to the response.